techniques/aria/ARIA22.html

```diff
@@ -26,13 +26,13 @@ <h2>Examples</h2>
 			<section class="example">
 				<h3>Including a search results message</h3>
 				<p>After a user presses a Search button, the page content is updated to include the results of the search, which are displayed in a section below the Search button. The change to content also includes the message <samp>"5 results returned"</samp> near the top of this new content. This text is given an appropriate role for a status message. A screen reader will announce "5 results returned".</p>
-<pre xml:space="preserve"><code class="language-html">&lt;div role="status" aria-atomic="true"&gt;5 results returned.&lt;/div&gt;</code></pre>
+<pre><code>&lt;div role="status" aria-atomic="true"&gt;5 results returned.&lt;/div&gt;</code></pre>
 				<p class="working-example"><a href="../../working-examples/aria-role-status-searchresults/">Working example: role=status on search results</a></p>
 			</section>
 			<section class="example">
 				<h3>Updating the shopping cart status</h3>
-				<p>After a user presses an Add to Shopping Cart button, content near the Shopping Cart icon updates to read "1 items". The container for this text (in this case a <code class="language-html">&lt;p&gt;</code>) is marked with the role of <code class="language-html">status</code>. Because it adds visual context, the shopping cart image &#8212; with succinct and accurate <code class="language-html">alt</code> text &#8212; is also placed in the container. Due to the <code class="language-html">aria-atomic</code> value, a screen reader will announce "Shopping cart, six items".</p>
-<pre xml:space="preserve"><code class="language-html">&lt;p role="status" aria-atomic="true"&gt;
+				<p>After a user presses an Add to Shopping Cart button, content near the Shopping Cart icon updates to read "1 items". The container for this text (in this case a <code>&lt;p&gt;</code>) is marked with the role of <code>status</code>. Because it adds visual context, the shopping cart image &#8212; with succinct and accurate <code>alt</code> text &#8212; is also placed in the container. Due to the <code>aria-atomic</code> value, a screen reader will announce "Shopping cart, six items".</p>
```

Comment:
```suggestion
				<p>After a user presses an Add to Shopping Cart button, content near the Shopping Cart icon updates to read "1 items". The container for this text (in this case a <code>&lt;p&gt;</code>) is marked with the <code>role="status"</code>. Because it adds visual context, the shopping cart image &#8212; with succinct and accurate <code>alt</code> text &#8212; is also placed in the container. Due to the <code>aria-atomic</code> value, a screen reader will announce "Shopping cart, six items".</p>
```